tasK_1.py

```diff
@@ -0,0 +1,27 @@
+"""
+Задача 1: Напишите программу, которая получает целое число и возвращает его шестнадцатеричное
+строковое представление. Функцию hex используйте для проверки своего результата.
+"""
+
+SYSTEM: int = 16
+result: str = "" 
+
+while True:
+    number = input("Введите любое число: ")
+    if not number.isdigit():
+        print("Попробуйте еще...")
+        continue
+    break
```

Comment:
Чтобы не использовать `break` и `continue` одновременно сделать и так:
```suggestion
    if number.isdigit():      
        break
    print("Попробуйте еще...")
```